backend/Tasque.BLL/Options/AmazonS3Options.cs

```diff
@@ -0,0 +1,11 @@
+﻿
+namespace Tasque.Core.BLL.Options
+{
+    public class AmazonS3Options
+    {
+        public string AwsAccessKey { get; set; } = null!;
+        public string AwsSecretAccessKey { get; set; } = null!;
+        public string BucketName { get; set; } = null!;
+        public string Region { get; set; } = null!;
+    }
```

Comment:
Make sure you and Danil will have the same config, as we will use one AWS account for everything